php/libraries/NDB_Page.class.inc

```diff
@@ -683,6 +683,7 @@ class NDB_Page
                   $baseurl . '/js/jquery/jquery-ui-1.10.4.custom.min.js',
                   $baseurl . '/js/jquery.dynamictable.js',
                   $baseurl . '/js/jquery.fileupload.js',
+                  $baseurl . '/js/jquery/timepicker/jquery.timepicker.js'
```

Comment:
missing comma breaks entirety of loris

Reply:
Ah, darn, was trying to sneak that one in there!